### PR TITLE
test against multiple Cython versions

### LINKED ISSUE #407: Support Cython 0.28.x

### PR DESCRIPTION
**Pull request by kmaehashi**
Use different version of supported Cython versions in each test.
Fixes #407.

### CODE REVIEW

**Comment by okuta** [comment]
The test was failed. Please check test result.

**Comment by kmaehashi** [comment]
Fixed.

File: run_test.py [hunk 1]
```diff
@@ -166,7 +166,7 @@
             'cudnn': 'cudnn7-cuda91',
             'nccl': 'nccl2.1-cuda91',
             'requires': [
-                'setuptools', 'cython==0.26.1', 'numpy<1.10', 'scipy<0.19',
+                'setuptools', 'cython==0.28.2', 'numpy<1.10', 'scipy<0.19',
```
Comment (okuta) [inline run_test.py]: `No matching distribution found for cython==0.28.2`

Comment (kmaehashi) [reply run_test.py]: Fixed!

**Comment by okuta** [comment]
jenkins, test this please

**Comment by okuta** [comment]
LGTM!

**Pull request merged**